I am not quick sure why the `Dependency Tree Diff` job [fails](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Android/29416/workflows/c3e47515-ce77-400a-9ba4-19bb93de21ec/jobs/119539) with the `task not found` exception, I tried a couple of things but nothing worked. Maybe this happens because the configuration needs to end up in `trunk` before the script can start working, due to the fact that we are switching to `targetBranch` at some point, I am not sure...

@kidinov any ideas? 🤔 🙏 

```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'dependencyTreeDiffCommentToGitHub' not found in root project 'project'.
```

--- a/tools/dependency-tree-diff/dependency-tree-diff.sh
+++ b/tools/dependency-tree-diff/dependency-tree-diff.sh
@@ -11,6 +11,8 @@ DEPENDENCY_TREE_VERSION="1.2.0"
 git config --global user.email '$( git log --format='%ae' $CIRCLE_SHA1^! )'
 git config --global user.name '$( git log --format='%an' $CIRCLE_SHA1^! )'
 
+currentBranch=$(git rev-parse --abbrev-ref HEAD)
+
 prNumber="${CIRCLE_PULL_REQUEST##*/}"
 githubUrl="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$prNumber"
 githubResponse="$(curl "$githubUrl" -H "Authorization: token $GITHUB_API_TOKEN")"
@@ -43,6 +45,7 @@ fi
 chmod +x dependency-tree-diff.jar
 ./dependency-tree-diff.jar $TARGET_BRANCH_DEPENDENCIES_FILE $CURRENT_TARGET_BRANCH_DEPENDENCIES_FILE > $DIFF_DEPENDENCIES_FILE
 
+git checkout "$currentBranch"
 if [ -s $DIFF_DEPENDENCIES_FILE ]; then
   echo "There are changes in dependencies of the project"
   cat "$DIFF_DEPENDENCIES_FILE"

